src/main/java/org/snomed/aag/data/services/ProjectAcceptanceCriteriaService.java

```diff
@@ -322,6 +324,7 @@ public boolean incrementIfComplete(ProjectAcceptanceCriteria projectAcceptanceCr
 
         if (allCriteriaItemsComplete && branchProjectLevel) {
             // New entry to get new creation date.
+            LOGGER.info("Project will be promoted; incrementing iteration.");
```

Comment:
This would be more helpful log message if there was some identifying information for the project the service is updating.

Reply:
Thanks, I have made that change.